tests/models/language.py

```diff
@@ -28,3 +34,64 @@ def test_language_specialchars_uniqueness():
     language.specialchars = u" Čč Ḍḍ Ɛɛ Ǧǧ Ɣɣ  Ḥḥ Ṣṣ ṬṬṭ Ẓẓ Ţţ Ṛṛ‌Ṛṛ‌"
     language.save()
     assert language.specialchars == u" ČčḌḍƐɛǦǧƔɣḤḥṢṣṬṭẒẓŢţṚṛ‌"
+
+
+@pytest.mark.django_db
+def test_language_iso_fullname(afrikaans):
+    new_fullname = "FULLNAME"
+    fullname = afrikaans.fullname
+    afrikaans.fullname = new_fullname
+    afrikaans.save()
+    assert afrikaans.fullname == new_fullname
+
+    settings.LANGUAGE_CODE = "cy"
```

Comment:
not sure if you need settings here, but if you do use the fixture rather than import